napari/components/_tests/test_layers_list.py

```diff
@@ -6,6 +6,15 @@
 from napari.components import LayerList
 from napari.layers import Image
 
+try:
+    import npe2  # noqa: F401
+
+    BUILTINS = 'napari'
+    SVG = 'napari-svg'
+except ImportError:
+    BUILTINS = 'builtins'
+    SVG = 'svg'
+
```

Comment:
in npe2, the plugin name should be the python package name. A consequence is that the npe2 versions of these plugins has a different name. Tests that rely on plugin lookup by name need the new name. Hence this.

See napari/npe2#57 for a proposed mechanism to let plugins keep their old names for the purposes of these lookups.